src/harness/sas_interface.py

```diff
@@ -404,4 +404,44 @@ def assertValidResponseFormatForApprovedGrant(self, grant_response):
       Nothing. It asserts if something about the response is broken/not per
       specs. Assumes it is dealing with an approved request.
     """
-    pass
\ No newline at end of file
+    pass
+
+  @abc.abstractmethod
+  def assertRegistered(self, registration_request,
+                       conditional_registration_data=None):
+    """Register a list of devices.
+
+    Quickly register N devices, assert registration SUCCESS, get CBSD IDs.
+    Includes injection of FCC IDs and conditional registration data.
+
+    Args:
```

Comment:
I don't think we want to pass the entire config file: each config file may have its own naming scheme for the parameters. For example, consider the PPA tests (recently proposed by Red and agreed to in WG4) in which they want to be able to specify a set of CBSDs that's 40 km away from the PPA and another set which is closer than 40 km. The easiest config for that would have two registration request variables, one for each category.

I think we want the helper functions to always assert/assume success as that gives us a nice tradeoff between complexity/easy of understanding and # of times the function can be reused. We could rename to something like "assertSuccessfullyRegistered" if you think that would be clearer?